backend/src/main/kotlin/xyz/elitese/ehrenamtskarte/importer/dataqa/FreinetDataFilter.kt

```diff
@@ -0,0 +1,10 @@
+package xyz.elitese.ehrenamtskarte.importer.dataqa
+
+import xyz.elitese.ehrenamtskarte.importer.types.FreinetAcceptingStore
+
+object FreinetDataFilter {
+    fun filterOutInvalidEntries(freinetAcceptingStores : List<FreinetAcceptingStore>) : List<FreinetAcceptingStore> {
```

Comment:
Are we sure that these should be filtered out? Can you post a list of partners which have invalid data?

Maybe we should transform the data to be valid instead of filtering the entries out.

Reply:
Currently i think we have less than 10 invalid entries, so not that urgent. Most of them should be fixable, however some are missing the category id which could be a problem. I think we need a concept for broken data later anyway and we also need to search for duplicates in the database etc. I think a separate user story/technical task would be good